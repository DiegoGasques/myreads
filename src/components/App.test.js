import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";

import BookApp from "./app";

it("Should match BookApp component snapshot", () => {
  const wrapper = shallow(<BookApp />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
