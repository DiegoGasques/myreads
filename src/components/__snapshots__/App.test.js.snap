// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match BookApp component snapshot 1`] = `
<ContextProvider
  value={
    Object {
      "state": Object {
        "books": Array [],
      },
      "updateBookStatus": [Function],
    }
  }
>
  <BrowserRouter>
    <div
      className="app"
    >
      <Route
        exact={true}
        path="/"
        render={[Function]}
      />
      <Route
        component={[Function]}
        path="/search"
      />
    </div>
  </BrowserRouter>
</ContextProvider>
`;
